en/micro-integrator/docs/references/connectors/smpp-connector/smpp-connector-config.md

```diff
@@ -165,15 +165,15 @@ To use the SMPP connector, add the `<SMPP.init>` element in your configuration b
        <td>Optional</td>
     </tr>
     <tr>
-       <td>distinationAddress</td>
+       <td>destinationAddress</td>
```

Comment:
can't change this. Original implementation carries the word 'distination' by mistake. Can't change it to maintain backward compatibility.